apps/server/src/utils/controllers.utils.ts

```diff
@@ -32,3 +32,35 @@ export function initializeRequestAndRetrieveProfile(req: Request): { userLike: U
   }
   return { userLike, successCallback };
 }
+
+/**
+ * Util that create package.
+ * @param req The user request.
+ * @returns any as { packageLike: Map; successCallback: () => void }
+ */
+export function initializeRequestAndRetrieveImagePackage(req: Request): {
+  packageLike: Record<string, string>;
```

Comment:
Update the type of package like
```suggestion
  packageLike: Partial<IPackage>;
```